storage_service/locations/api/resources.py

```diff
@@ -467,6 +469,7 @@ def prepend_urls(self):
         ]
 
     def obj_create(self, bundle, **kwargs):
+        related_package_uuid = bundle.data.get('related_package_uuid')
         bundle = super(PackageResource, self).obj_create(bundle, **kwargs)
```

Comment:
I don't think the bundle is populated until after the `super` call.  This should be moved afterwards.